pkgs/tools/misc/ntfy-sh/default.nix

```
@@ -1,5 +1,11 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ pkgs, lib, nodejs, stdenv, buildGoModule, fetchFromGitHub, debianutils, mkdocs, python3, python3Packages }:
```

We have a convention where we put `lib` before anything else:

```suggestion
{ lib, pkgs, nodejs, stdenv, buildGoModule, fetchFromGitHub, debianutils, mkdocs, python3, python3Packages }:
```